composer.json

```diff
@@ -34,9 +34,12 @@
         "monolog/monolog": "^1.0",
         "php-http/guzzle6-adapter": "^1.0.0",
         "php-http/mock-client": "^0.3.2",
+        "phpunit/phpunit": "^5.7 || ^6.0",
         "symfony/process": "^2.3 || ^3.0 || ^4.0",
         "symfony/http-kernel": "^2.3 || ^3.0 || ^4.0",
-        "phpunit/phpunit": "^5.7 || ^6.0"
+        "symfony/lock": "^3.4 || ^4.0",
+        "symfony/cache": "^3.4 || ^4.0",
```

Comment:
@Toflar do we need the lock and cache as direct dependency? guess not, we only had that for stability weakening? the store brings them and its the store that has the dependency, not directly us?

Reply:
They're not needed. They're `dev` requirements anyway and I guess it's a leftover from the times where the store implementation was part of this PR and not a separate lib yet :)

Reply:
Was also probably needed during dev/beta of lock :)